setup.cfg

```diff
@@ -18,13 +19,30 @@ classifiers =
     Programming Language :: Python :: 3.8
 
 [options]
-include-package-data = true
+include_package_data = true
+packages = gltf
+python_requires = >=3.6
+install_requires =
+    panda3d >= 1.10.8
+    panda3d-simplepbr >= 0.6
+    importlib-metadata >= 1.0 ; python_version < "3.8"
+
+[options.entry_points]
+console_scripts =
+    gltf2bam = gltf.cli:main
+gui_scripts =
+    gltf-viewer = gltf.viewer:main
+    octodns-report = octodns.cmds.report:main
+    octodns-sync = octodns.cmds.sync:main
```

Comment:
OctoDNS?

Reply:
copy-paste error from the example I found